classes/Domain/Services/TalkEmailer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Domain\Services;
+
+use Swift_Mailer;
+use Swift_Message;
+
+class TalkEmailer
```

Comment:
I don't really see the purpose of that class. It's a thin wrapper around the `Swift_Mailer` without any real logic. Why does the upgrade to Swiftmailer 6 make it necessary to introduce that class?